test/e2e/memtierd.test-suite/no-policy/n4c16/test02-tracker-finder/code.var.sh

```diff
@@ -0,0 +1,69 @@
+memtierd-setup
+
+# Create swap
+zram-install
+zram-swap off
+zram-swap 2G
+
+MEME_BS=1G
+memtierd-meme-start
+
+# 1. Find the 1G address range from meme (pages -pid MEME_PID -pr 20)
+# 2. Verify that the whole range is found as a single range when PagemapBitPresent is ignored (no pagemap bit scan)
+# 4. swap -out -pid MEME_PID -ranges (a-range-in-the-middle) and verify swapped out range is found with PagemapBitPresent:false
+# 5. Verify to find ... -- swapout-start and swapout-end -- all-end ranges with PagemapBit:true
+# 6. Verify to find full start-end region when PagemapBitPresent is ignored also when the middle range is swapped out.
+
+MEMTIERD_YAML=""
+memtierd-start
+memtierd-command "pages -pid ${MEME_PID} -pr 50"
+echo "# Find the address range of the ${MEME_BS} allocation"
+PARSED_AR=$(awk 'BEGIN{FS="[-( ]+";midoffset=1024*1024*50;midsize=1024*1024*100}
+/[0-9a-f]+-/{start=$1;end=$2;size=$3;
+if (size>1024*1024*1023){
+    startmid=strtonum("0x"$1)+midoffset
```

Comment:
When I tried to run the e2e tests, I got an err:

memtierd> # Find the address range of the 1G allocation
awk: line 9: function **strtonum** never defined

error: failed to find the start and end addresses of the 1G block
host> scp -o StrictHostKeyChecking=No -o ControlMaster=auto -o ControlPath=/home/jing/vms/data/n4c16-ubuntu-22.04-containerd/ssh -o ControlPersist=30 ubuntu@172.17.0.2:memtierd*.output.txt "/home/jing/go/src/github.com/intel/memtierd/test/e2e/memtierd.test-suite/no-policy/n4c16/test02-tracker-finder/output/"
The VM, Kubernetes and cri-resmgr are left running. Next steps:
- Login VM:     ssh ubuntu@172.17.0.2
- Stop VM:      govm stop n4c16-ubuntu-22.04-containerd
- Delete VM:    govm delete n4c16-ubuntu-22.04-containerd
Test verdict: FAIL

Tests summary:
FAIL no-policy/n4c16/test02-tracker-finder

Reply:
wow, great finding!

Reply:
Switched to awkless implementation. Thanks for your help with that!